Run tests after `make upgrade-requirements`

dannyroberts opened the pull request.

dannyroberts committed:
Run make upgrade-requirements

dannyroberts committed:
Disable python 2 travis tests

dannyroberts committed:
Django 2: Always use named args in models.Index

dannyroberts committed:
Django 2: Switch django.core.urlresolvers => django.urls

https://stackoverflow.com/questions/43139081/importerror-no-module-named-django-core-urlresolvers

emord commented on requirements-python3/dev-requirements.txt:
```diff
@@ -65,7 +65,7 @@ django-transfer==0.4
 django-user-agents==0.3.2
 django-websocket-redis==0.5.2
-django==1.11.22
+django==2.0.13
```
I think you'll likely want to pin this better in requirments.in. I can't imagine we'd want to upgrade all of these and django at the same time (unless minor version)

I had been running this once a week for a while, but been putting it off until after the deprecation of py2

emord replied:
nevermind i guess. I see from your branch that you know how reckless you're being. carry on

dannyroberts committed:
Unpin and upgrade djangorestframework 3.5.4 => 3.10.2 (latest)

dannyroberts committed:
Temporarily limit travis to a singal job (until that is passing)

dannyroberts committed:
Django 2: RegexURLResolver => URLResolver

still falls back to importing RegexURLResolver (to support Django 1.11
as well).

dannyroberts committed:
Django 2: Upgrade django-angular (djangular => djng)

dannyroberts committed:
Django 2: Upgrade celery to 4.3.0

dannyroberts committed:
Django 2: remove defunct egg template loader

added 8 years ago as a dependency for the now-removed djtables:
https://github.com/dimagi/commcare-hq/commit/64c751ed2afa298d7e0565569348d30133068159

dannyroberts committed:
Run `make upgrade-requirements` again

dannyroberts committed:
Django 2: upgrade django-otp and django-two-factor-auth

dannyroberts committed:
Run `make upgrade-requirements` again

dannyroberts committed:
Django 2: Upgrade redis libraries
